### ISSUE TITLE
Does Sextant work with the trial version of VoiceAttack?

### ISSUE THREAD

**Issue by Simm0** [Stickymaddness/Sextant#4]
For try as I might, I can't get it to work.  If it does work with the trial version, then two possible causes come to mind:
i) I am using Win10 and have a later version of .NET (I tried to install 4.6.1 but could not as an error occurred informing me that already had a later version)
ii) Win username is different to that shown within File Manager - actual is my email address, File Manager abbreviates it to 5 chars..but have tried both in settings.json to no avail - so probably can discount this one
VoiceAttack (run as admin) does not give a voice line, nor show plugin enabled...
Any suggestions greatly appreciated.

**Comment by Stickymaddness** [comment]
Yes, Sextant works with the trial version of VoiceAttack - the only difference with the trial version is a limit of 10 commands.

What is the full path to your settings.json?

**Comment by Simm0** [comment]
Hi

Okay, thanks.

Please see below for the path. Let me know if you need anything else




_______________

Mark Sim
+44 7973 658 377

**Comment by Stickymaddness** [comment]
I don't see the path in your reply.

**Comment by Simm0** [comment]
Hi,

Perhaps I am not sure of what you are asking me then as I thought the top
of the screenshot would suffice (c:\program files
(x86)\voiceattack\apps\sextant)

If that is not it, please let me know

Thanks

_______________

Mark Sim
+44 7973 658 377
